types/pdfmake/index.d.ts

```diff
@@ -175,7 +175,7 @@ declare module 'pdfmake/build/pdfmake' {
         compress?: boolean;
         header?: TDocumentHeaderFooterFunction;
         footer?: TDocumentHeaderFooterFunction;
-        content: string | Content;
+        content: Array<string | Content>;
```

Comment:
Hi @ZaLiTHkA, thanks for submitting the PR. I originally created the type definitions based on the demo files on the pdfmake website.

Your fix is probably correct and necessary, as I did not use the type definitions in a real world project yet.

One remark though: We should be consistent with the way of writing arrays.
I suggest to change line 178 to `content: (string | Content)[]` to match the style in line 156.

Reply:
At first, I did actually adjust line 178 to be `(string | Content)[]`, but then [this test result](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/480299304#L2478) said I couldn't do it like that. :/

At first I thought this was just going to be a quick update, extending the type definitions to cover a few extra cases that weren't being covered... But now it's turning out to be a pretty big mission.

I'm a bit busy with actual work stuff at present, so I'll dig into this one when I get home this evening instead.

Reply:
The log does not load properly at the moment and says "We couldn't find the repository DefinitelyTyped/DefinitelyTyped" (which is weird 😄). I will check back later.